src/org/labkey/targetedms/outliers/CUSUMOutliers.java

```diff
@@ -517,16 +520,17 @@ public int hashCode()
     }
 
 
-    private Map<String, Integer> processEachOutlier(boolean groupByGuideSet, Map<String, Integer> countObj, int guideSetId, Set<String> sampleFiles, String sampleFile)
+    private Map<String, Integer> processEachOutlier(boolean groupByGuideSet, Map<String, Integer> countObj, int guideSetId, Set<String> sampleFiles, String sampleFile, String sampleFileString)
```

Comment:
These arguments need some additional explanation. How do sampleFile and sampleFileString relate. Part of my confusion is that they're both of type String.